bin/main.rb

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env ruby
+
+require_relative '../lib/players'
+require_relative '../lib/logic'
+
+# Instantiate the Player class
+player1 = Player.new('X')
+player2 = Player.new('O')
+running = true
+
+while running
+
+  # Welcome
+  puts '-----------------------'
+  puts '  ♥  ♥        ♥  ♥'
+  puts ' ♥     ♥   ♥       ♥'
+  puts '  ♥      ♥        ♥'
+  puts '   ♥  Welcome to ♥'
+  puts '    ♥  Tic Tac  ♥'
+  puts '      ♥  Toe  ♥'
+  puts '        ♥   ♥'
+  puts '          ♥'
+  puts '----------------------'
+  puts 'X will be player one, what is your name?'
+  player1.name = gets.chomp
+  puts "Hello #{player1.name}, you are now the X"
+  puts 'O will be player two, what is your name? (Please insert a nickname)'
+  player2.name = gets.chomp
+  puts "Hello #{player2.name}, you are now the O"
+  puts ' --------------------------------------------------------------- '
+  puts
+  puts
+  puts "Ready to play #{player1.name} and #{player2.name}? Let's go!"
```

Comment:
I can't see this puts on the terminal, it just skips to the game. 

![CavernousEnergeticBluetonguelizard-size_restricted](https://user-images.githubusercontent.com/64173189/104788291-f93d8400-575f-11eb-8b5a-77768eb3dccc.gif)